### PR TITLE
Harden TestProbeSpec: increase short timeouts

### PR DESCRIPTION
**Pull request by chbatey**
Refs #26681

### CODE REVIEW

**Review by raboof — APPROVED** [review]

File: akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/TestProbeSpec.scala [hunk 1]
```diff
@@ -37,7 +39,7 @@ class TestProbeSpec extends ScalaTestWithActorTestKit with WordSpecLike {
     "allow probing for actor stop when actor already stopped" in {
       val probe = TestProbe()
       val ref = spawn(Behaviors.stopped)
-      probe.expectTerminated(ref, 100.millis)
+      probe.expectTerminated(ref, shortDuration)
```
Comment (patriknw) [inline akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/TestProbeSpec.scala]: This was the failing test. I see no reason to use a short duration at all here. `probe.expectTerminated(ref)` should do it.

Comment (chbatey) [reply akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/TestProbeSpec.scala]: :+1:

Comment (chbatey) [reply akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/TestProbeSpec.scala]: The other timeouts were tight so i'll leave the other changes in

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**